src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx

```diff
@@ -1,9 +1,8 @@
 import React, { useEffect, useState } from 'react';
 import { func, object } from 'prop-types';
 import {
-  Button,
+  IconButton,
```

Comment:
Can you sort these imports? It's helpful having the items in the import objects sorted alphabetically